### PR TITLE
Clean missing galleries

### PR DESCRIPTION
**Pull request by WithoutPants**
Changed the clean operation so that it also checks and removes galleries that are no longer present. Changed the clean dialog message accordingly.

### CODE REVIEW

File: pkg/manager/manager_tasks.go [hunk 1]
```diff
@@ -506,7 +514,26 @@ func (s *singleton) Clean() {
...
 			wg.Add(1)
 
-			task := CleanTask{Scene: *scene}
+			task := CleanTask{Scene: scene}
+			go task.Start(&wg)
+			wg.Wait()
+		}
+
+		for _, gallery := range galleries {
+			s.Status.incrementProgress()
```
Comment (bnkai) [inline pkg/manager/manager_tasks.go:523]: s.Status.incrementProgress() 
shouldnt this be something like s.Status.setProgress(len(scenes)+j, total) ?
where j the for index for gallery
If you want to use increment shouldn't the Task total be set first and increment also used for scenes?

Comment (WithoutPants) [reply pkg/manager/manager_tasks.go:523]: I can do that. It was just done to not use `len`. The calls to `setProgress` set the total.

Comment (bnkai) [reply pkg/manager/manager_tasks.go:523]: i didn't notice that the status total was updated that way.
you can leave it as is then

**Comment by bnkai** [comment]
Not related to your changes
Can you replace in `pkg/manager/exclude_files.go` function `matchFile`
`logger.Infof("No exclude patterns in config.")`
with
`return false` ?
I noticed that it pollutes the log if no exclude pattern is set in the config.

Everything tests out ok

**Pull request merged**